tracker/data/env.py

```diff
@@ -25,4 +25,4 @@
 SCANNERS = ["pshtt", "sslyze"]
 
 # Used if --lambda is enabled during the scan.
-LAMBDA_WORKERS = 900
+LAMBDA_WORKERS = 700
```

Comment:
I'm curious, were you finding issues doing 900 at a time vs 700?

Reply:
Yeah, I was getting it set up at the end of last week and did a full lambda run for the first time, and was getting timed out at 900 (which I know is odd, seeing as it should allow 1000 concurant).

I bumped it down and the issues went away. I had meant to ping you if you had experienced similar, but it slipped my mind.

Reply:
I think @jsf9k has experienced something like this before, though I haven't noticed it for me. I definitely noticed it at 1000, which is why I dropped it to 900.